IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -238,34 +240,34 @@ - (nullable instancetype)initWithGroup:(nullable NSString *)keychainGroup
             keychainGroup = [[NSBundle mainBundle] bundleIdentifier];
         }
 
-        // TODO: Mac-specific MSIDKeychainUtil to support teamId: https://identitydivision.visualstudio.com/DevEx/_workitems/edit/660959
-        if (MSIDKeychainUtil.teamId)
+        NSString *teamId = [self.class teamId];
```

Comment:
This breaks all of our tests since our test app is not signed.

Can you please provide an internal method "initForTests" without this check and we will extract it in a test extension?

Reply:
oh, looks like you already have something like this